{{cookiecutter.repository_name}}/data/example.py

```diff
@@ -30,12 +32,21 @@ def augment(self, augmenter):
         return self.replace(image=augmenter.augment(image=self.image))
 
 
+def text_(draw, text, x, y, fill="black", outline="white", size=12):
+    font = ImageFont.load_default()
```

Comment:
size is not used here and you can't change size when using the default font,
maybe we should include some font in the library?

Reply:
Fixed in https://github.com/nextml-code/pytorch-lantern-template/pull/23 :)